### ISSUE TITLE
Signing out due to cloud provider issues

### ISSUE THREAD

**Issue by mubashirrbaig** [raivo-otp/ios-application#268]
**Describe the bug**
I have attached the issue screenshot, despite signing in again, I am getting this error repeatedly everyday for last some days.

![IMG_4204](https://github.com/raivo-otp/ios-application/assets/26192731/30ec6642-c5b7-41b7-aa0f-0c4b37454b4d)

**To Reproduce**
Don't know

**Expected behavior**
It does not repeatedly signout or have syncing issues with the cloud provider.

**Screenshots**
 attcahed above

**Smartphone (please complete the following information):**
 - Device: iPhone 13
 - OS: iOS 16.5.1 (20F75)
 - Version : 1.4.21

**Additional context**
-

**Comment by Churator** [comment]
I see. this sometimes too, now again.
no need to sign out actually, killing the app and starting again fix this.

**Comment by BingeCode** [comment]
Yep this is the only bug I have found with Raivo OTP as well. Would be nice to find the root cause and fix it.
